docs/gallery/general/read_basics.py

```diff
@@ -69,62 +68,47 @@
 #   :alt: selecting a folder on dandi
 #   :align: center
 #
-# Stream the data
-# ^^^^^^^^^^^^^^^
 #
-# Next, we will demonstrate how to stream the data from the DANDI archive without
-# having to download it to your machine.
-# Streaming data requires having HDF5 installed with the ROS3 (read-only S3) driver.
-# You can install from `conda-forge <https://conda-forge.org>`_ using ``conda``.
-# You might need to first uninstall a currently installed version of ``h5py``.
-#
-# .. code-block:: bash
-#
-#    $ pip uninstall h5py
-#    $ conda install -c conda-forge "h5py>=3.2"
-#
-# We can access the data stored in an S3 bucket using the DANDI API,
-# which can be installed from pip:
-#
-# .. code-block:: bash
-#
-#    $ pip install -U dandi
-#
-# .. seealso::
-#     You can learn more about streaming data in the :ref:`streaming` tutorial.
-#
-# Then, we will use the :py:class:`~dandi.dandiapi.DandiAPIClient` to obtain the S3 URL that points to the NWB File
-# stored in S3. We will need the identifier of the dataset (``dandiset_id``) and the path
-# to the NWB File.
-# We can read these from the DANDI archive URL where ``dandiset_id`` is "000004" and
-# file is located in "sub-P11HMH" folder.
-
-
-dandiset_id = "000004"
-filepath = "sub-P11HMH/sub-P11HMH_ses-20061101_ecephys+image.nwb"
-with DandiAPIClient() as client:
-    asset = client.get_dandiset(dandiset_id, "draft").get_asset_by_path(filepath)
-    s3_path = asset.get_content_url(follow_redirects=1, strip_query=True)
-
-####################
-# Using NWBHDF5IO
-# ---------------
+# Opening an NWB file with NWBHDF5IO
+# ----------------------------------
 #
 # Reading and writing NWB data is carried out using the :py:class:`~pynwb.NWBHDF5IO` class.
 # :py:class:`~pynwb.NWBHDF5IO` reads NWB data that is in the `HDF5 <https://www.hdfgroup.org/solutions/hdf5/>`_
 # storage format, a popular, hierarchical format for storing large-scale scientific data.
 #
-# The first argument to the constructor of :py:class:`~pynwb.NWBHDF5IO` is the ``file_path`` -
-# this can be the path that points to the downloaded file on your computer or
-# it can be an S3 URL.
-#
-# Use the ``read`` method to read the data into a :py:class:`~pynwb.file.NWBFile` object.
+# The first argument to the constructor of :py:class:`~pynwb.NWBHDF5IO` is the ``file_path``. Use the ``read`` method to
+# read the data into a :py:class:`~pynwb.file.NWBFile` object.
 
-# Open the file in read mode "r", and specify the driver as "ros3" for S3 files
-io = NWBHDF5IO(s3_path, mode="r", driver="ros3")
+filepath = "path/to/file.nwb"
+# Open the file in read mode "r",
+io = NWBHDF5IO(filepath, mode="r", load_namespaces=True)
```

Comment:
The tests for this tutorial fail here because ``"path/to/file.nwb"`` is not a valid file. If we don't want the tutorial to download the file from DANDI, then it would be good to have backup option to ensure the tutorial works correctly, e.g.., by setting the filepath to a simple, dummy NWB file if the user does not set the path manually to the file they donwloaded from DANDI. 




```
Traceback (most recent call last):
  File "/home/runner/work/pynwb/pynwb/test.py", line 128, in __run_example_tests_helper
    _import_from_file(script)
  File "/home/runner/work/pynwb/pynwb/test.py", line 72, in _import_from_file
    return imp.load_source(os.path.basename(script), script)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda/envs/ros3/lib/python3.11/imp.py", line 172, in load_source
    module = _load(spec)
             ^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 721, in _load
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/runner/work/pynwb/pynwb/docs/gallery/general/read_basics.py", line 84, in <module>
    io = NWBHDF5IO(filepath, mode="r", load_namespaces=True)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda/envs/ros3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/pynwb/pynwb/src/pynwb/__init__.py", line 236, in __init__
    super().load_namespaces(tm, path, file=file_obj, driver=driver)
  File "/usr/share/miniconda/envs/ros3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda/envs/ros3/lib/python3.11/site-packages/hdmf/backends/hdf5/h5tools.py", line 165, in load_namespaces
    open_file_obj = cls.__resolve_file_obj(path, file_obj, driver)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda/envs/ros3/lib/python3.11/site-packages/hdmf/backends/hdf5/h5tools.py", line 140, in __resolve_file_obj
    file_obj = File(path, 'r', **file_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda/envs/ros3/lib/python3.11/site-packages/h5py/_hl/files.py", line 567, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, fcpl, swmr=swmr)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda/envs/ros3/lib/python3.11/site-packages/h5py/_hl/files.py", line 231, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 106, in h5py.h5f.open
FileNotFoundError: [Errno 2] Unable to synchronously open file (unable to open file: name = 'path/to/file.nwb', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
```

Reply:
ok I'll add download to the tutorial

Reply:
@oruebel ok, I added some lines to download the file progamatically. It's a small file so I don't think it is going to cause any bottlenecks

Reply:
Thanks!